cadc-rest/src/main/java/ca/nrc/cadc/rest/InlineContentHandler.java

```diff
@@ -101,7 +101,7 @@
     Content accept(String name, String contentType, InputStream inputStream)
         throws InlineContentException, IOException, ResourceNotFoundException;
 
-    public static class Content
+    class Content
```

Comment:
This should be static (so it is clear that lifecycle goes beyond that if the InlineContentHandler that created it. 

And because it is something that implementations create with new it should be marked public. I know that technically works with package access but public reflects the intent better. It also allows one to write an InlineContentHandler impl that goes beyond one class (I think only the class that "implements InlineContentHandler" can see this otherwise.

Reply:
Inner Classes of interfaces are `public` `static` already.

Reply:
OK.